osu.iOS/Main.cs

```diff
@@ -9,6 +9,9 @@ public class Application
         // This is the main entry point of the application.
         static void Main(string[] args)
         {
+            // required to initialise native SQLite libraries on some platforms.
+            SQLitePCL.Batteries_V2.Init();

```

Comment:
isn't this already done in the DbContexts?

Reply:
The netstandard branch moved this out of DbContext :/

Reply:
Hmm i see.. so where is it on desktop?

Reply:
Also why can it not be in DbContext?

Reply:
https://github.com/ppy/osu/blob/ios/osu.Desktop/Program.cs#L19

I think I moved it to initially reduce references in osu.Game when redoing the csproj's, should be possible to move it back.